e2e/wdio/core/tests/menu.e2e-spec.ts

```diff
@@ -0,0 +1,120 @@
+import { MenuPo } from '../pages/menu.po';
+import { checkElementScreenshot,
+    click, 
+    mouseHoverElement, 
+    saveElementScreenshot, 
+    addIsActiveClass,
+    focusElement, 
+    getElementArrayLength,
+    pause,
+    waitForElDisplayed,
+    elementDisplayed,
+    doesItExist} from '../../driver/wdio';
+
+describe('Menu tests', function() {
+    const menuPage = new MenuPo();
+    const { menuButtonsArr, 
+        btnMobileMenu, 
+        menuItemsArr, 
+        dialogMobileMenu,
+        closeDialogMobileMenu,
+        dialogMenuAddonArr,
+        dialogMenuItemsArr,
+        dialogBtnBack } = menuPage;
+
+    beforeAll(() => {
+        menuPage.open();                              
+    }, 1);
+
+    it('should check RTL and LTR orientation', () => {
+        menuPage.checkRtlSwitch();
+    });
+
+    it('should check visual regression for all examples', () => {
```

Comment:
please make rtl check and visual regression the last test cases in the spec and do a page refresh before visual regression case.